docs/connect/python/index.md

```diff
@@ -0,0 +1,119 @@
+(connect-python)=
+# Python
+
+:::{div} sd-text-muted
+Connect to CrateDB and CrateDB Cloud from Python.
+:::
+
+Individual adapters, clients, and drivers offer specific features for specific
+needs of your application, so please read this enumeration carefully, to
+select the optimal adapter client or driver for your purposes.
+
```

Comment:
Maybe not within this patch yet, but revolving specifically around Python, DuckDB also provides an excellent example how to conceive a corresponding section across different driver domains and connectivity options.

- https://duckdb.org/docs/stable/clients/python/overview

Maybe this can spark some interest to support us in improving our Python presentation correspondingly? Please send patches! 🙏